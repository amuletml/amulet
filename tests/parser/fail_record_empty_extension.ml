let a = { 0 with }
