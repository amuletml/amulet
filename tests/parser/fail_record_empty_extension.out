fail_record_empty_extension.ml[1:18 ..1:18]: error
  Unexpected }, expected identifier
  │ 
1 │ let a = { 0 with }
  │ 
